com.unity.shadergraph/Editor/Drawing/Views/MaterialGraphView.cs

```diff
@@ -1308,22 +1317,21 @@ internal static void InsertCopyPasteGraph(this MaterialGraphView graphView, Copy
             }
         }
 
-        private static void ClampNodesWithinView(MaterialGraphView graphView, IEnumerable<AbstractMaterialNode> nodeList)
+        private static void ClampNodesWithinView(MaterialGraphView graphView, IEnumerable<IRectInterface> rectList)
```

Comment:
This change just makes the algorithm work on an interface to get/set the rect of an object so the same code can run on nodes and sticky notes.